.github/workflows/osv-scanner-scan.yml

```diff
@@ -0,0 +1,34 @@
+# This file is adapted from https://github.com/google/osv-scanner
+
+
+name: OSV-Scanner Scan
+
+on:
+  pull_request:
```

Comment:
We don't want this trigger.

Reply:
This was needed to test this PR itself, i can remove it now